src/operators/mod.rs

```diff
@@ -4,29 +4,37 @@ use pyo3::prelude::*;
 
 use crate::pyo3_extensions::{TdPyAny, TdPyCallable, TdPyIterator};
 use crate::with_traceback;
+use crate::log_func;
+
+use log::debug;
 
 // These are all shims which map the Timely Rust API into equivalent
 // calls to Python functions through PyO3.
 
 pub(crate) fn map(mapper: &TdPyCallable, item: TdPyAny) -> TdPyAny {
+    debug!("{}, mapper:{:?}, item:{:?}", log_func!(), mapper, item);
```

Comment:
It would be nice to log out epoch as well in all of these so you can flip a single logging switch and see a complete trace of execution. But that would be a bigger change because it would require changing the Timely operators we use and re-jiggering some of the shim functions, so I don't think we need to do it now. Just a note for posterity!